networking_aci/plugins/ml2/drivers/mech_aci/driver.py

```diff
@@ -210,6 +210,10 @@ def _get_clearable_phys_doms(self, network, local_segment, host_config):
 
         for other_binding in other_bindings:
             _, other_binding_host_config = self._host_or_host_group(other_binding.host)
+            if not other_binding_host_config:
+                LOG.warning("No config available / invalid binding host %s, segment %s with profile %s in network %s",
+                            other_binding.host, other_binding.segment_id, other_binding.profile, network['id'])
```

Comment:
Are you sure, `other_binding` alwas has all those attributes?

Reply:
Portbinding.profile was missing from the SQL query, fixed that. All attributes should now be present as a result of the query, or did you see anything else?

Reply:
I just wasn't sure about the model at all. Was trying to see if you checked it :)